tools/webidl_binder.py

```diff
@@ -406,8 +406,19 @@ def render_function(class_name, func_name, sigs, return_type, non_pointer, copy,
   if not js_impl: continue
   implements[name] = [js_impl[0]]
 
+nodeHeight = {}
+for child, parent in implements.iteritems():
```

Comment:
this depends on the order of iteration, doesn't it? or is there some reason that this order is guaranteed to be correct? (i.e., `nodeHeight.get` might succeed or fail based on the order)